src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -25,28 +29,51 @@ type Props = {
     translations?: Translations,
 };
 
-const ActiveState = ({
-    currentUser,
-    items,
-    onCommentDelete,
-    onTaskDelete,
-    onTaskEdit,
-    onTaskAssignmentUpdate,
-    onVersionInfo,
-    translations,
-    getAvatarUrl,
-    getUserProfileUrl,
-    getMentionWithQuery,
-    mentionSelectorContacts,
-}: Props): React.Node => (
-    <ul className="bcs-activity-feed-active-state">
-        {items.map((item: any) => {
-            const { type, id, versions, permissions } = item;
+class ActiveState extends React.Component<Props> {
+    cache: any;
+
+    list: any;
+
+    constructor() {
+        super();
+        this.cache = new CellMeasurerCache({
+            fixedWidth: true,
+        });
+    }
+
+    componentDidUpdate() {
+        this.cache.clearAll();
+        this.list.forceUpdateGrid();
+    }
 
-            switch (type) {
-                case 'comment':
-                    return (
-                        <li key={type + id} className="bcs-activity-feed-comment">
+    onEditChange = () => {
+        this.cache.clearAll();
+        this.list.forceUpdateGrid();
+    };
+
+    renderActivityCard = ({ key, index, parent, style }: any) => {
+        const {
+            items,
+            currentUser,
+            getAvatarUrl,
+            getUserProfileUrl,
+            onCommentDelete,
+            translations,
+            getMentionWithQuery,
+            mentionSelectorContacts,
+            onTaskAssignmentUpdate,
+            onTaskDelete,
+            onTaskEdit,
+            onVersionInfo,
+        } = this.props;
+
+        const item: any = items[index];
+        const { type, id, versions, permissions } = item;
+        switch (type) {
+            case 'comment':
+                return (
+                    <CellMeasurer cache={this.cache} columnIndex={0} key={key} parent={parent} rowIndex={index}>
```

Comment:
I'm not super familiar with react-virtualized, but if I'm reading the [docs](https://github.com/bvaughn/react-virtualized/blob/master/docs/Grid.md) correctly, you might be able to pass `deferredMeasurementCache` to `List` to avoid passing `this.cache` around.